packages/arb-bridge-peripherals/contracts/tokenbridge/libraries/ITransferAndCall.sol

```diff
@@ -13,7 +13,7 @@ interface IERC677 is IERC20Upgradeable {
     event Transfer(address indexed from, address indexed to, uint256 value, bytes data);
 }
 
-interface IERC677Receiver {
+interface ITransferAndCallReceiver {
```

Comment:
I think we should add a comment here about the expected behaviour for reverts / return value.

Reply:
this sufficient? not sure if there's more we want to say about reverts